airflow/cli/cli_parser.py

```diff
@@ -182,6 +182,13 @@ def add_to_parser(self, parser: argparse.ArgumentParser):
     ("--limit",),
     help="Return a limited number of records")
 
+# next_execution
+ARG_NUM_EXECUTIONS = Arg(
+    ("-n", "--num-executions"),
+    default=1,
+    type=int,
```

Comment:
Should we not prohibit negative or zero numbers? https://stackoverflow.com/a/25296177